### PR TITLE
Add a completion sub-command to generate completion for bash

### PR DESCRIPTION
**Pull request by vdemeester**
Signed-off-by: Vincent Demeester <vincent@sbr.pm>

### CODE REVIEW

File: cmd/docker-app/completion.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	"github.com/spf13/cobra"
+)
+
+func completionCmd(dockerCli command.Cli, rootCmd *cobra.Command) *cobra.Command {
+	return &cobra.Command{
+		Use:   "completion",
+		Short: "Generates bash completion scripts",
+		Long: `To load completion run
+
+. <(bitbucket completion)
```
Comment (chris-crone) [inline cmd/docker-app/completion.go]: bitbucket? :)

File: cmd/docker-app/completion.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+To configure your bash shell to load completions for each session add to your bashrc
+
+# ~/.bashrc or ~/.profile
+. <(bitbucket completion)
```
Comment (chris-crone) [inline cmd/docker-app/completion.go]: bitbucket? :)

Comment (vdemeester) [reply cmd/docker-app/completion.go]: lol :joy:

**Comment by silvin-lubecki** [comment]
And what about zsh completion 😅

**Comment by vdemeester** [comment]
@silvin-lubecki and `fish` completion too :stuck_out_tongue: Let's start with bash and let's iterate from it :angel:

**Pull request merged**